src/style.css

```
@@ -0,0 +1,62 @@
+
+body {
+  margin: 0;
+}
+
+.stars_block {
+  margin: auto;
+  width: auto;
+  height: auto;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+  
+ 
+
+.stars .stars__star {
```

A lot of spaces. And you don't really need to increase the specificity here
```suggestion
}

.stars__star {
```